File: example/AnyJob.Java.Demo/src/main/java/demo/Static.java
package demo;

import java.util.ArrayList;
import java.util.List;

public class Static {
    public static int add(int num1, int num2) {
        return num1 + num2;
    }

    public static String concat(String a, String b) {
        return a + b;
    }

    public static void hello(String name) {
        System.out.println("Hello," + name);
    }

    public static List<Person> merge(List<Person> persons, Person other) {
        List<Person> result = new ArrayList<Person>();
        if (persons != null) {
            result.addAll(persons);
        }
        if (other != null) {
            result.add(other);
        }
        return result;
    }
}

File: example/AnyJob.NetCore.Demo/Static.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace AnyJob.NetCore.Demo
{
    public class Static
    {
        public static int Add(int num1, int num2)
        {
            return num1 + num2;
        }

        public static string Concat(string a, string b)
        {
            return a + b;
        }
        public static void Hello(string name)
        {
            Console.WriteLine($"Hello,{name}");
        }

        public static List<PersonInfo> Merge(List<PersonInfo> persons, PersonInfo other)
        {
            var result = new List<PersonInfo>();

            result.AddRange(persons ?? Enumerable.Empty<PersonInfo>());
            if (other != null)
            {
                result.Add(other);
            }
            return result;
        }
    }
}
